shared/build.gradle.kts

```diff
@@ -67,6 +100,42 @@ android {
     }
 }
 
-dependencies {
-    testImplementation(Dependency.JUNIT)
+val buildKonfigProperties = rootDir.loadGradleProperties("buildKonfig.properties")
+
+buildkonfig {
+    packageName = "co.nimblehq.ic.kmm.suv"
+
+    defaultConfigs {
+        buildConfigField(
+            STRING,
+            "CLIENT_ID",
+            buildKonfigProperties.getProperty("CLIENT_ID")
+        )
+        buildConfigField(
+            STRING,
+            "CLIENT_SECRET",
+            buildKonfigProperties.getProperty("CLIENT_SECRET")
+        )
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            buildKonfigProperties.getProperty("DEFAULT_BASE_URL")
+        )
```

Comment:
we have BASE_URL for prod and staging env already, why do we need to have 1 more DEFAULT_BASE_URL?

Reply:
This is a limitation of [this library](https://github.com/yshrsmz/BuildKonfig), the default config is required. However, I will improve this by use `STAGING_BASE_URL` as default :D

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/86/commits/267561ece685f282ba111bf3132e937820df7d04